.github/dependabot.yml

```
@@ -16,3 +16,10 @@ updates:
   commit-message:
     prefix: patch
   rebase-strategy: disabled
+- package-ecosystem: "github-actions"
+  directory: ".github/workflows"
```

```suggestion
  directory: "/"
```
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot

---

I wonder why they don't let you go the other way around of specifying the correct directory.